examples/step-89/doc/results.dox

```diff
@@ -72,9 +72,9 @@ for overlapping triangulations. For this the intersection requests are of dimens
 The cells which are intersected with the intersection requests are of dimension $dim$. For the
 simple case of non-conforming interfaces it would be sufficient to compute the intersections
 between $dim-1$ and $dim-1$ entities. Furthermore, the lambda could be adapted, such that cells are
-only marked if they are connected to a certain boundary ID (in this case, e.g. 99) instead of
-marking every cell that is <i>not</i> connected to the opposite boundary ID (in this case, e.g. 98).
-Marking less cells can reduce the setup costs significantly.
+only marked if they are connected to a certain boundary ID (in this case, e.g., 99) instead of
+marking every cell that is <i>not</i> connected to the opposite boundary ID (in this case, e.g., 98).
+Marking fewer cells can reduce the setup costs significantly.
 
-Note that for in-homogeneous material in this procedure is questionable, since it is not clear which
+Note that the use of in-homogeneous materials in this procedure is questionable, since it is not clear which
```

Comment:
Was there before, but I would have a slight preference towards
```suggestion
Note that the use of inhomogeneous materials in this procedure is questionable, since it is not clear which
```